src/chartLibraries/dygraph/crosshair.test.js

```
@@ -0,0 +1,277 @@
+import crosshair from "./crosshair"
+
+describe("crosshair", () => {
+  let mockChartUI, mockDygraph, mockChart, mockCanvas, mockCtx
```

ESLint

[error] 'mockCanvas' is defined but never used.

(no-unused-vars)